tests/test_other.py

```diff
@@ -660,6 +660,11 @@ def test_cmake_bitcode_static_libraries(self):
   def test_cmake_emscripten_version(self):
     self.run_process([EMCMAKE, 'cmake', test_file('cmake/emscripten_version')])
 
+  def test_cmake_find_stuff(self):
+    # Ensure that zlib exists in the sysroot
+    self.run_process([EMCC, test_file('hello_world.c'), '-sUSE_ZLIB'])
+    self.run_process([EMCMAKE, 'cmake', test_file('cmake/find_stuff')])
```

Comment:
How about adding this new test to run on windows too?

Reply:
Oh we already run `other.test_cmake*` on windows!